mlir/include/Quantum/IR/QuantumInterfaces.td

```diff
@@ -129,7 +129,7 @@ def QuantumGate : OpInterface<"QuantumGate", [QuantumOperation]> {
         >,
         InterfaceMethod<
             "Return all operands which are considered controlling output qubit values.",
-            "mlir::ValueRange", "getCtrlQubitResults"
+            "mlir::ResultRange", "getCtrlQubitResults"
```

Comment:
There is an method further up called `getNonCtrlQubitResults` that needs updating as well.